cfgov/data_research/models.py

```diff
@@ -176,3 +179,20 @@ def __str__(self):
 
     class Meta:
         ordering = ['name']
+
+
+class MortgageMetaData(models.Model):
+    """
+    Metadata values, stored as json, to supplement display of mortgage charts.
+    """
+    name = models.CharField(max_length=255)
+    json_value = models.TextField(blank=True)
```

Comment:
@chosak Rather than add another dependency to this PR, I'd like to defer the use of django-jsonfields to a future PR.